### PR TITLE
ST-1051: Fixed issue with date field in Case Linking feature

### PR DESCRIPTION
**Pull request by sjamihmcts**
### Change description ###
Fixed issue with date field in Case Linking feature

### CODE REVIEW

**Review by Shailesh-walke — APPROVED** [review]

File: ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/type/CaseLink.java [hunk 1]
```diff
@@ -22,7 +22,7 @@ public class CaseLink {
   private List<ListValue<LinkReason>> reasonForLink;
 
   @JsonProperty("CreatedDateTime")
-  private LocalDate createdDateTime;
+  private LocalDateTime createdDateTime;
```
Comment (rajaram-nagarajan) [inline ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/type/CaseLink.java:25]: @sjamihmcts - Is it better to add the date time format here? (remember we had some issues without the formatter)

Comment (sjamihmcts) [reply ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/type/CaseLink.java:25]: ok got you Rajaram. Will add the JsonFormat as well.

Comment (linusnorton) [reply ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/type/CaseLink.java:25]: yeah that's a good shout

Comment (sjamihmcts) [reply ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/type/CaseLink.java:27]: Added the JsonFormat attribute.

**Review by rajaram-nagarajan — APPROVED** [review]
LGTM

**Pull request merged**